chore: added observable application (#158)
vaadin/observability-kit: observability-kit-demo/README.md

> # Observability KIT demo Application
> 
> Observability KIT demo application is a Vaadin application with Spring Boot from start.vaadin.com
> It contains features to test slowness, errors and other functionality for using the
> vaadin observability extension with OpenTelemetry javaAgent.

Reviewer: ```suggestion
# Observability Kit demo application

This demo application is a Vaadin application from start.vaadin.com tuned to
test slowness, errors and other issues that are detected by Observability Kit.
```

> 
> 
> # Running the application with OTel Java Agent
> 
> There are default properties and a shell script to run NewRelic, Dynatrace and Jaeger+prometheus.
> Just add the correct path to the javaAgent into the shell script.
> 
> [NOTE]
> NewRelic properties file needs the API key from the saas server.
> +
> Dynatrace properties need the Authorization api key generated on the saas server.
> +
> Jaeger targets default ports for jaeger and prometheus.

Reviewer: Is this still relevant now that we provide the Docker setup?

```suggestion
```
